app/src/utils/Utils.tsx

```diff
@@ -478,3 +478,40 @@ export const getRandomHexColor = (seed: number, min = 100, max = 170): string =>
  * @return {*}  {value is T}
  */
 export const isDefined = <T,>(value: T | undefined | null): value is T => value !== undefined && value !== null;
+
+/**
+ * Creates a GeoJSON point from a set of coordinates and associated properties.
+ *
+ * @param latitude - The latitude coordinate.
+ * @param longitude - The longitude coordinate.
+ * @returns A GeoJSON point
+ */
+export const createGeoJSONPoint = (latitude: number, longitude: number): Point => {
```

Comment:
Maybe these helper functions could be moved to its own utils file. utils/spatial or utils/geoJson